app/workers/janitor_worker.rb

```diff
@@ -7,6 +7,7 @@ def perform
     return unless ThreeScale.config.janitor_worker_enabled
 
     PurgeOldUserSessionsWorker.perform_async
-    PurgeOldEventsWorker.perform_async
+    PurgeStaleObjectsWorker.perform_later(EventStore::Event.name, DeletedObject.name)
+    true
```

Comment:
Is this `true` required? 

I think the `assert JanitorWorker.new.perform` works if we return any value different from `nil`

Reply:
the `JanitorWorker.new.perform` removing this true returns `nil` 😞 so i can return any truthy value other than true but i am not sure if it is worth it 😄

Reply:
maybe you can test if `PurgeStaleObjectsWorker` was enqueued.. 

or leave the true 😂 not a big deal haha